README.md

```diff
@@ -18,6 +18,281 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
+- Retrieve a Self Assessment Tax Calculation
+
+Added calculation type parameter `in-year` and `intent-to-finalise` for the following endpoint:
+- Trigger Tax Calculation
+- List Calculations
+
+Added calculation type parameter `intent-to-amend` for the tax year 2025-26 onwards for the following endpoint:
+- Trigger Tax Calculation
+- List Calculations
+
+Added new tax year specefic scehema for tax year 2023-24 onwards for the following endpoint:
+- List Calculations
+
+Added errors 
+`RULE_OUTSIDE_AMENDMENT_WINDOW`, 
+`RULE_DECLARATION_NOT_RECEIVED`, 
+`INVALID_CALCULATION_TYPE`, 
+`FORMAT_CALCULATION_TYPE`
+for the following endpoint:
+- Trigger Tax Calculation
+
+Added error `RULE_CALCULATION_TYPE_NOT_ALLOWED` when a calculation type does not apply to the tax year requested for the following endpoint:
+- Trigger Tax Calculation
+- List Calculations
+
+Added tax year specific schemas for tax year 2025-26 onwards for the following endpoint:
+- List Calculations
+##### Changed
+
+Changed `submissionPeriods` to `submissionPeriod`, its type to object and `periodId` to `submissionId` for tax years 2025-26 onwards for the following endpoint:
+- Retrieve a Self Assessment Tax Calculation
+
+Updated CalculationType enum to:
+"IY" => `in-year`,
+"IF" => `intent-to-finalise`,
+"IA" => `intent-to-amend`,
+"DF" => `declare-finalisation`,
+"CA" => `confirm-amendment`
+for the following endpoint:
+- Retrieve a Self Assessment Tax Calculation
+##### Removed
+Removed fields `metadata.intentToSubmitFinalDeclaration`, `metadata.finalDeclaration` and `metadata.finalDeclarationTimestamp` from the response for the following endpoint:
+- Retrieve a Self Assessment Tax Calculation
+
+#### Business Source Adjustable Summary (BSAS) API
+
+API version 6.0 is now available in Sandbox, with the following change.
+
+##### Added
+Added object `ukProperty` for tax year 2025-26 onwards for the following endpoint:
+- Submit UK Property Accounting Adjustments
+##### Changed
+Renamed `nonFurnishedHolidayLet` to `ukProperty` for tax years before 2025-26 for the following endpoint:
+- Submit UK Property Accounting Adjustments
+
+Renamed object `nonFurnishedHolidayLet` to `foreignProperty` for tax years before 2025-26 for the following endpoint:
+- Submit Foreign Property Accounting Adjustments
+
+Rename `typeOfBusiness` enum `uk-property-non-fhl` to `uk-property` for tax year before 2025-26 for the following endpoints:
+- Trigger a Business Source Adjustable Summary
+- List Business Source Adjustable Summaries
+
+Change enum values in request query parameters and response for field `typeOfBusiness` to `self-employment`, `foreign-property` and `uk-property` for tax year 2025-26 for the following endpoint:
+- List Business Source Adjustable Summaries
+
+Renamed enum value for field typeOfBusiness from uk-property-non-fhl to uk-property for tax year before 2025-26 for the following endpoint:
```

Comment:
are uk-property-non-fhl (and uk-property) missing the backticks to make them "code-words"?